symengine/derivative.cpp

```diff
@@ -670,6 +641,46 @@ class DiffImplementation
             }
         }
     }
+
+    static RCP<const Basic> diff(const LeviCivita &self,
+                                 const RCP<const Symbol> &x)
+    {
+        return diff0(self, x);
+    }
+
+    template <typename T>
+    static RCP<const Basic> diff0(const T &self, const RCP<const Symbol> &x)
```

Comment:
@isuruf I initially tried to do that, by first trying `Basic` and then `Function`, but always got an error. These classes do not have the function `create`.
```
/home/shikhar/symengine/symengine/derivative.cpp: In static member function ‘static Teuchos::RCP<const SymEngine::Basic> SymEngine::DiffImplementation::diff(const SymEngine::Function&, const Teuchos::RCP<const SymEngine::Symbol>&)’:
/home/shikhar/symengine/symengine/derivative.cpp:679:56: error: ‘const class SymEngine::Function’ has no member named ‘create’
                                Derivative::create(self.create(v), {v[i]}), m)));
```